supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1,10 +1,901 @@
+
+[discrete]
+[[jboss-eap-built-in-messaging]]
+==== JBoss EAP built-in messaging (noun)
+*Description*: "JBoss EAP built-in messaging" is an acceptable term for referring to the built-in messaging system in JBoss EAP. Other acceptable terms are "built-in messaging" and "JBoss EAP messaging".
+
+*Use it*: yes
+
+*Incorrect forms*: ActiveMQ, ActiveMQ Artemis
+
+*See also*: xref:built-in-messaging[built-in messaging], xref:jboss-eap-messaging[JBoss EAP messaging]
+
+[discrete]
+[[jboss-eap-messaging]]
+==== JBoss EAP messaging (noun)
+*Description*: "JBoss EAP messaging" is an acceptable term for referring to the built-in messaging system in JBoss EAP. Other acceptable terms are "built-in messaging" and "JBoss EAP built-in messaging".
+
+*Use it*: yes
+
+*Incorrect forms*: ActiveMQ, ActiveMQ Artemis
+
+*See also*: xref:built-in-messaging[built-in messaging], xref:jboss-eap-built-in-messaging[JBoss EAP built-in messaging]
+
+[discrete]
+[[jca]]
+==== jca (noun)
+*Description*: The "jca" subsystem is used to configure settings for the Jakarta EE Connector Architecture (JCA) container. Write in lower case as one word. See xref:jca-heading[JCA] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: JCA, Jca
+
+*See also*: xref:jca-heading[JCA]
+
+[discrete]
+[[jca-heading]]
+==== JCA (noun)
+*Description*: Use "JCA" when referring to the xref:jca[jca] subsystem in titles and headings. Write in uppercase as one word. See xref:jca[jca] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Jca, jca
+
+*See also*: xref:jca[jca]
+
+[discrete]
+[[jdr]]
+==== jdr (noun)
+*Description*: The "jdr" subsystem is used to gather diagnostic data to support troubleshooting. Write in lower case as one word. See xref:jdr-heading[JDR] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: JDR, Jdr
+
+*See also*: xref:jdr-heading[JDR]
+
+[discrete]
+[[jdr-heading]]
+==== JDR (noun)
+*Description*: Use "JDR" when referring to the xref:jdr[jdr] subsystem in titles and headings. Write in uppercase as one word. See xref:jdr[jdr] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Jdr, jdr
+
+*See also*: xref:jdr[jdr]
+
+[discrete]
+[[jgroups]]
+==== jgroups (noun)
+*Description*: The "jgroups" subsystem is used to configure protocol stacks and communication mechanisms for servers in a cluster. Write in lower case as one word. See xref:jgroups-heading[JGroups] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: JGROUPS, JGroups
+
+*See also*: xref:jgroups-heading[JGroups]
+
+[discrete]
+[[jgroups-heading]]
+==== JGroups (noun)
+*Description*: Use "JGroups" when referring to the xref:jgroups[jgroups] subsystem in titles and headings. Capitalize the 'J' and 'G' and write the rest of the word in lower case. See xref:jgroups[jgroups] entry for the correct usage in general text.
```

Comment:
`write the rest of the word in lower case` seems like it's implied already, right? Delete. Ditto for subsequent entries.

Reply:
Fixed